File: 202_搜尋/TestGraphII/Node.cs
using System;
namespace lesson202_01
{
    public class Node
    {
        private int data;
        private int weight;
        private Node link;
        public Node()
        {
            
        }
        public Node(int data, int weight)
        {
            this.data = data;
            this.weight = weight;
            this.link = null;
        }
        public int GetData()
        {
            return this.data;
        }
        public int GetWeight()
        {
            return this.weight;
        }
        public Node GetLink()
        {
            return this.link;
        }
        public void SetLink(Node link)
        {
            this.link = link;
        }
    }
}

File: 107_雜湊/TestDictionaryII/Node.java

public class Node
{
	private int key;
	private String value;
	private Node link;
	public Node(int key, String val)
	{
		this.key = key;
		this.value = val;
		this.link = null;
	}
	public String getValue()
	{
		return this.value;
	}
	public Node getLink()
	{
		return this.link;
	}
	public void setLink(Node link)
	{
		this.link = link;
	}
}
